junction/proposals/migrations/0027_auto_20200502_0540.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9 on 2020-05-02 00:10
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("proposals", "0026_auto_20200323_2010"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="historicalproposal",
+            name="video_url",
+            field=models.URLField(
+                blank=True,
+                default="",
+                help_text="A short 1-2 mins video link about your talk",
+            ),
+        ),
+        migrations.AddField(
+            model_name="proposal",
+            name="video_url",
+            field=models.URLField(
+                blank=True,
+                default="",
+                help_text="A short 1-2 mins video link about your talk",
+            ),
+        ),
+    ]
```

Comment:
Just trying to understand why there are two migrations?

Reply:
Yes, these changes are tested. The historical data is added from `django-simple-history` which I guess provides kind of an audit log.

![image](https://user-images.githubusercontent.com/500628/80864139-5a8af900-8c9e-11ea-85a2-9d33d2472795.png)